From: Hylke Bons <hylkebons@gmail.com>
Date: Fri, 13 Jul 2012 15:24:06 +0200
Subject: [PATCH] linux statusicon: Fix statusicon animation. Closes #849

---
 SparkleShare/Linux/SparkleStatusIcon.cs |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/SparkleShare/Linux/SparkleStatusIcon.cs b/SparkleShare/Linux/SparkleStatusIcon.cs
index dc71709..6f02fd1 100644
--- a/SparkleShare/Linux/SparkleStatusIcon.cs
+++ b/SparkleShare/Linux/SparkleStatusIcon.cs
@@ -290,10 +290,10 @@ namespace SparkleShare {
         {
             this.animation_frames     = new Gdk.Pixbuf [5];
             this.animation_frames [0] = SparkleUIHelpers.GetIcon ("process-syncing-i", 24);
-            this.animation_frames [0] = SparkleUIHelpers.GetIcon ("process-syncing-ii", 24);
-            this.animation_frames [0] = SparkleUIHelpers.GetIcon ("process-syncing-iii", 24);
-            this.animation_frames [0] = SparkleUIHelpers.GetIcon ("process-syncing-iiii", 24);
-            this.animation_frames [0] = SparkleUIHelpers.GetIcon ("process-syncing-iiiii", 24);
+            this.animation_frames [1] = SparkleUIHelpers.GetIcon ("process-syncing-ii", 24);
+            this.animation_frames [2] = SparkleUIHelpers.GetIcon ("process-syncing-iii", 24);
+            this.animation_frames [3] = SparkleUIHelpers.GetIcon ("process-syncing-iiii", 24);
+            this.animation_frames [4] = SparkleUIHelpers.GetIcon ("process-syncing-iiiii", 24);
         }
 
 
-- 
1.7.10.2.1.g8c77c3c

